ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -523,6 +523,14 @@ VAV_HWZTC_CHWZTC_DTM:
     - CHWZTC
     - DTM
 
+VAV_MSVO_PDSCV:
```

Comment:
What exactly is the motion sensor used for? Does the damper position control differently if motion is detected?

Reply:
If motion is detected, the damper is opened to discharge air, if not, the damper is closed.

Reply:
Done.